Send empty dictionary instead of null for iot queries

sdb9696 opened the pull request:
Fixes: #1120

UDP query from TAPO app is
```
{"system":{"get_sysinfo":{}},"cnCloud":{"get_info":{}},"smartlife.iot.common.cloud":{"get_info":{}},"smartlife.cam.ipcamera.cloud":{"get_info":{}}}
```

rytilahti reviewed (approved):
LGTM and works on all my connected iot devices, my tapo led strips are disconnected but I don't doubt it wouldn't work with them (seeing the klap discovery query got also updated), too.

sdb9696 commented:
> LGTM and works on all my connected iot devices, my tapo led strips are disconnected but I don't doubt it wouldn't work with them (seeing the klap discovery query got also updated), too.

Was thinking about that and the discovery query shouldn't be in the `klaptransport` at all, it probably pre-dates the splitting of the transports from the protocols, I have removed the `DISCOVERY_QUERY` variable from that module.
